app/src/components/TransferArtifact.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import Button from 'react-bootstrap/Button';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import { FormControlProps } from 'react-bootstrap/FormControl';
+import Modal from 'react-bootstrap/Modal';
+
+interface TransferArtifactProps {
+  drizzle: any;
+  drizzleState: any;
+  tokenId: any;
```

Comment:
Any for tokenId seems a bit excessive

Also you can extend the drizzled interface to get the right types for drizzle and drizzleState

Reply:
I'm not sure how to extend the drizzled interface. Would this involve specifying the types for all the fields of drizzle and drizzleState?